libraries/AP_HAL_ChibiOS/GPIO.cpp

```diff
@@ -73,6 +79,13 @@ void GPIO::init()
     uint8_t chan_offset = 0;
 #if HAL_WITH_IO_MCU
     if (AP_BoardConfig::io_enabled()) {
+        uint8_t GPIO_mask = 0;
+        for (uint8_t i=0; i<8; i++) {
+            if (SRV_Channels::is_GPIO(i)) {
+                GPIO_mask |= 1 << i;
```

Comment:
I prefer 1U << i so it is clearly unsigned. Doesn't matter in this case, but good habit